### PR TITLE
Minion command line starter for issue #6111

### PR DESCRIPTION
**Pull request by jtao15**
Minion command line starter for issue #6111

### CODE REVIEW

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotMinion.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+/**
+ * Simple shim so that jps -l shows PinotMinion instead of PinotAdministrator
+ */
+public class PinotMinion {
```
Comment (snleee) [inline pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotMinion.java:29]: admin/PinotMinion.java

Comment (jtao15) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotMinion.java:29]: For other components e.g. server. We can starts with both `bin/start-server.sh` and `bin/pinot-admin.sh`. I create this so users can starts with `bin/start-minion.sh` also.

Comment (snleee) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotMinion.java:29]: Ah i see, that makes sense.

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java [hunk 2]
```diff
@@ -184,7 +184,7 @@ protected void startMinion(@Nullable Map<String, PinotTaskExecutorFactory> taskE
     FileUtils.deleteQuietly(new File(Minion.DEFAULT_INSTANCE_BASE_DIR));
     try {
       _minionStarter =
-          new MinionStarter(ZkStarter.DEFAULT_ZK_STR, getHelixClusterName(), new PinotConfiguration());
+          new MinionStarter(getHelixClusterName(), ZkStarter.DEFAULT_ZK_STR, new PinotConfiguration());
```
[the remark this answers is not in the document]
Comment (jtao15) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java:187]: I switched the order of the first 2 parameters for `MinionStarter` to be consistent with other starters constructors.

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartMinionCommand.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      return false;
+    }
+  }
+
+  private Map<String, Object> getMinionConf()
+      throws ConfigurationException {
+    if (_configFileName != null) {
+      return PinotConfigUtils.readConfigFromFile(_configFileName);
+    }
+    return PinotConfigUtils.generateMinionConf(_minionPort);
```
[the remark this answers is not in the document]
Comment (jtao15) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartMinionCommand.java:108]: Yes, I think we can do that.

**Review by snleee — APPROVED** [review]

**Pull request merged**